main.html

```diff
@@ -1,77 +1,255 @@
+  The principles are:
+</p>
+
+<ul>
+  <li>Voluntary Code Contribution</li>
+  <li>Prioritized Mentorship</li>
+</ul>
+
+<p>
+  Let's take a look at each of these principles in more detail.
+</p>
+
+<h3>Voluntary Code Contribution</h3>
+
+<p>
+  The first word <em>Voluntary</em> means that engagement in InnerSource from both the guest and host teams occurs of their own free will.
+  The guest team voluntarily donates code to the host team and the host team voluntarily accepts it.
+  This opt-in nature means that each team needs to be certain that their involvement adds value to the others' objectives.
+  Never is a host team required to accept a contribution that isn't in ultimate alignment with their overall mission.
+  Never is a guest team required to submit a contribution that doesn't ultimately further their own mission and priorities.
+</p>
+
+<p>
+  The word <em>Code</em> emphasizes that the collaboration between guest and host goes all the way to the code.
+  Guest involvement in opening issues, updating requirements, fixing docs, etc. is good, but collaboration needs to reach as far as submitting code to achieve all of the benefits that we've discussed.
+</p>
+
+<h3>Prioritized Mentorship</h3>
+
+<p>
+  <em>Mentorship</em> from host team to guest team is a key aspect of InnerSource.
+  Contributors on guest teams are upleveled so that they understand enough about the host team system to change it successfully.
+  In the process of doing so, they come to better understand the host team system as a general consumer.
+  They can use it more effectively and help other consumers to do so as well. 
+</p>
+
+<p>
+  It's critical that this mentorship for contributors is <em>Prioritized</em> by the host team.
+  Trusted committer availability to mentor is made available to guest team contributors <em>at the time that they need it</em> as opposed to when it is convenient for the trusted committer and host team.
+</p>
+
+
+<h2>Conclusion</h2>
+
+<p>
+  In this Oriole, we've given an introduction to InnerSource.
+  InnerSource applies open source best practices and principles to internal software development. 
+  It gives an additional option to consumers when producing teams are not able to deliver a needed feature request.
+  Successful InnerSource involves a <strong>product owner</strong> and <strong>trusted committer</strong> from the <strong>host team</strong> as well as a <strong>contributor</strong> from the <strong>guest team</strong>.
+  Done effectively, InnerSource brings many benefits to both participating teams.
+  They key principles upon-which effective InnerSource works are <strong>voluntary code contribution</strong> and <strong>prioritized mentorship</strong>.
+</p>
+
+<p>
+  While this training contains a high-level overview of InnerSource, there are many more details useful in making InnerSource actually work for your team.
```

Comment:
This sentence made me think that it might be nice to open with 
> You should learn these things in this Oriole:
> What problem does innersourcing solve
> The two main principles of innersource
> Am I doing Innersource already?
> What a basic innersource pattern looks like. 
> Innersource anti-patterns

Something along those lines.  It'd be nice to know what I'm getting myself into at the outset and what I should expect to walk away with.

Reply:
ok! I'll comment on the main body of text...